plugin.video.retrospect/channels/channel.regionalnl/openregio/chn_openregio.py

```diff
@@ -0,0 +1,332 @@
+        self.liveUrls = None        # : a collection of live streams
+        self.recentUrl = None       # : the url for most recent items
+
+        if self.channelCode == "wosnl":
+            self.noImage = "wosnlimage.jpg"
+            self.mainListUri = "https://media.wos.nl/retrospect/wos/wos-index.json"
+            self.baseUrl = "https://media.wos.nl/retrospect/"
+            self.liveUrl = "https://media.wos.nl/stream/wos/live.php"
+            self.recentUrl = "https://media.wos.nl/retrospect/wos/wos-latest.json"
+
+        elif self.channelCode == "dtvnl":
+            self.noImage = "dtvnlimage.jpg"
+            self.mainListUri = "https://media.wos.nl/retrospect/dtvnieuws/dtvnieuws-index.json"
+            self.baseUrl = "https://media.wos.nl/retrospect/"
+            self.recentUrl = "https://media.wos.nl/retrospect/dtvnieuws/dtvnieuws-latest.json"
+            self.liveUrls = {
+                "Dtv Oss & Bernheze": "https://rrr.sz.xlcdn.com/?account=dtvmaasland&file=dtv1&type=live&service=wowza&output=playlist.m3u8&format=jsonp",
+                "Dtv Den Bosch": "https://rrr.sz.xlcdn.com/?account=dtvmaasland&file=dtv2&type=live&service=wowza&output=playlist.m3u8&format=jsonp",
+                "Dtv Uden": "https://rrr.sz.xlcdn.com/?account=dtvmaasland&file=dtv2&type=live&service=wowza&output=playlist.m3u8&format=jsonp",
+                "Mfm Radio": "https://rrr.sz.xlcdn.com/?account=dtvmaasland&file=mfm1&type=live&service=icecast&format=jsonp"
+            }
+
+        elif self.channelCode == "venlonl":
+            self.noImage = "venlonlimage.jpg"
+            self.mainListUri = "https://media.wos.nl/retrospect/omroepvenlo/omroepvenlo-index.json"
+            self.baseUrl = "https://media.wos.nl/retrospect/"
+            self.recentUrl = "https://media.wos.nl/retrospect/omroepvenlo/omroepvenlo-latest.json"
+
+        elif self.channelCode == "horstnl":
+            self.noImage = "horstnlimage.jpg"
+            self.mainListUri = "https://media.wos.nl/retrospect/omroephorstaandemaas/omroephorstaandemaas-index.json"
+            self.baseUrl = "https://media.wos.nl/retrospect/"
+            self.recentUrl = "https://media.wos.nl/retrospect/omroephorstaandemaas/omroephorstaandemaas-latest.json"
+
+        elif self.channelCode == "studio040":
+            self.noImage = "studio040image.jpg"
+            self.mainListUri = "https://media.wos.nl/retrospect/studio040/studio040-index.json"
+            self.baseUrl = "https://media.wos.nl/retrospect/"
+            self.recentUrl = "https://media.wos.nl/retrospect/studio040/studio040-latest.json"
+
+        else:
+            raise NotImplementedError("Channelcode '%s' not implemented" % (self.channelCode, ))
+
+        # setup the main parsing data
+        self._add_data_parser(self.mainListUri, json=True,
+                              name="Mainlist parser",
+                              preprocessor=self.add_other_items, match_type=ParserData.MatchExact,
```

Comment:
If his is shown in the UI it needs to be localised

Reply:
It is it not. Just in the logfile.